packages/jest-gas-reporter/package.json

```diff
@@ -12,8 +12,8 @@
     "prepare": "yarn build",
     "build": "yarn lint:check && tsc -b",
     "build:typescript": "tsc -b",
-    "lint:check": "eslint \"*/**/*.ts\" --cache",
-    "lint:write": "eslint \"*/**/*.ts\" --fix"
+    "lint:check": "eslint \"src/**/*.ts\" --cache",
```

Comment:
Why is this only checking `src` and `ts` files?
Do we not want linting for all files?